common/src/main/java/dev/lavalink/youtube/http/YoutubeOauth2Handler.java

```diff
@@ -289,6 +290,12 @@ public void applyToken(HttpUriRequest request) {
         }
     }
 
+    public void applyToken(HttpUriRequest request, String token) {
+        if (token != null && !token.isEmpty()) {
```

Comment:
this token empty check should prob be done in the caller instead to ensure that the other oauth token is being used if this one is empty